js/tests/components/song/list.spec.js

```diff
@@ -6,6 +6,8 @@ import { event } from '@/utils'
 import { songStore, queueStore } from '@/stores'
 import { playback } from '@/services'
 import { mock } from '@/tests/__helpers__'
+import { shallowMount } from "@vue/test-utils"
```

Comment:
Please prefer single quotes over double quotes in JS.

Reply:
Done. I suggest adding the [quotes](https://eslint.org/docs/rules/quotes) eslint rule. That way, it can be fixed automatically.